### DISCUSSION TITLE
Question about the "value_for_tiktoken"

### DISCUSSION THREAD

**Discussion by L4zyy** [camel-ai/camel#397] [General]
At https://github.com/camel-ai/camel/blob/master/camel/types/enums.py#L41

```
@property
    def value_for_tiktoken(self) -> str:
        return self.value if self is ModelType.STUB else "gpt-3.5-turbo"

```

The model_type to get the tiktoken is fixed to "gpt-3.5-turbo" unless we are using the stub model. I wonder if this is doing on purpose? Is it because the result are same for these model types? (which is <Encoding 'cl100k_base'>)

**Comment by lightaime** [comment]
@L4zyy thanks for spotting this bug. It seems to be added accidently by a recent PR:
https://github.com/camel-ai/camel/pull/376/files

I will fix it in this PR: https://github.com/camel-ai/camel/pull/399.
